extensions/azurecore/src/extension.ts

```diff
@@ -328,3 +331,6 @@ async function displayReloadAds(): Promise<boolean> {
 
 }
 
+async function displayDeprecated(): Promise<void> {
+	await vscode.window.showInformationMessage(loc.deprecatedOption);
```

Comment:
You probably shouldn't await this - if the user doesn't immediately respond to it then it'll hide and go into the tray but this will still be waiting for them to specifically x it away

`void vscode.window.showInformationMessage(loc.deprecatedOption);` should be enough

(nit side note - I don't think a function just for calling one other function is really worth the overhead)